cluster-state-service/internal/features/_test/gucumbertest__.go

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Please dont check this in. These files are generated during gucumber test runs and aren't automatically deleted if you abort the test run.